### PR TITLE
{math}[intel/2019b] statsmodels v0.9.0 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/6207ca6a2b52839620de8882af8158f5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7ba7b10d7f98c1153c3d75ec076f5ae9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2407.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/744213c36c68424c7d2951274fda46f4 for a full test report.

File: easybuild/easyconfigs/s/statsmodels/statsmodels-0.9.0-intel-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+exts_list = [
+    ('patsy', '0.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/patsy'],
+        'checksums': ['f115cec4201e1465cd58b9866b0b0e7b941caafec129869057405bfe5b5e3991'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/s/statsmodels'],
+        'checksums': ['6461f93a842c649922c2c9a9bc9d9c4834110b89de8c4af196a791ab8f42ba3b'],
+    }),
+]
```
Comment (lexming) [inline easybuild/easyconfigs/s/statsmodels/statsmodels-0.9.0-intel-2019b-Python-2.7.16.eb]:
```suggestion
exts_default_options = {'source_urls': [PYPI_SOURCE]}
exts_list = [
    ('patsy', '0.5.1', {
        'checksums': ['f115cec4201e1465cd58b9866b0b0e7b941caafec129869057405bfe5b5e3991'],
    }),
    (name, version, {
        'checksums': ['6461f93a842c649922c2c9a9bc9d9c4834110b89de8c4af196a791ab8f42ba3b'],
    }),
]
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/c4baabd460f531ecdd386f27e81c2794 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2fdab056a7d51a232b0036e8c187cf79 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**